### DISCUSSION TITLE
Customize alert evaluation behavior

### DISCUSSION THREAD

**Discussion by lex-pec** [grafana/grafana#63090] [Alerting]
Hello, everyone.

I use Grafana alerts not only for tracking the critical values of my panels, but also as some kind of reports to see the current values of my queries. To make a "report" I am putting such a condition in the settings that the alert state of my alert is always `Alerting`, plus I am using `Evaluate every` setting to customize how often it will be executed and sending a message. 

But it is better to configure the exact time of execution, like `at 6 PM every day` or `at 6 AM and at 7 PM every day`.
I am sure that this feature will find its practical application in solving other issues as well.

The feature may look like this:
<img width="400" alt="Снимок экрана 2023-02-08 в 13 28 28" src="https://user-images.githubusercontent.com/48240070/217517449-793cc631-7342-431e-8f0b-87433af47829.png">

**Comment by gillesdemey** [comment]
Hi! I think it's better to ask these types of questions in the [Prometheus user group](https://groups.google.com/g/prometheus-users) or the [Prometheus GitHub discussions](https://github.com/prometheus/prometheus/discussions).

**Reply by lex-pec** [reply]
Hello again.
But I am not a Prometheus user. The screenshot is taken on the alert configuration page in the Grafana UI.

**Reply by gillesdemey** [reply]
Sorry I should probably elaborate on this a bit; Grafana uses the same alert generator specification as Prometheus (and imports it into the project) to evaluate alerts.

Since Cron-style evaluation intervals are not supported by Prometheus we cannot support it in Grafana either.

**Reply by lex-pec** [reply]
Got it, thank you for the information!

**Comment by gelicia** [comment]
This discussion looks like it received an answer, so I will close it.
